apps/backend/src/events/events.controller.ts

```diff
@@ -17,8 +29,13 @@ export class EventsController {
   }
 
   @Get()
-  async findAll(): Promise<Event[]> {
-    return this.eventsService.findAll();
+  @ApiQuery({ name: 'skip', type: Number })
+  @ApiQuery({ name: 'take', type: Number })
+  async findAll(
+    @Query('skip', new DefaultValuePipe(0), ParseIntPipe) skip?: number,
+    @Query('take', new DefaultValuePipe(10), ParseIntPipe) take?: number
+  ): Promise<Event[]> {
+    return this.eventsService.findAll(take, skip);
   }
```

Comment:
the DefaultValuePipe is a nice addition, but please also add the required: false flag to the @ApiQuery-s, so that swagger will allow empty values (and then the default value will be used)